### ISSUE TITLE
`CustomerNumber` when creating new customer is ignored

### ISSUE THREAD

**Issue by alimony** [xalien10/pyfortnox#53]
```python
client = fortnox.Client(access_token="foo", client_secret="bar")
new_customer = client.customers.create(CustomerNumber=123, Name="Test Name")
print(new_customer.CustomerNumber)
'3096'
```
where 3096 is just the next in order, created automatically by Fortnox. I can confirm that calling the API directly with the same data properly creates a new object with the correct passed customer number and name.

**Comment by alimony** [comment]
I see that this, in general, happens because parameters in create methods are filtered using the `OPTS_KEYS_TO_PERSIST` class attribute. What is the reason for this filtering? I can't find anywhere in the Fortnox documentation (so far) that other parameters would not be accepted e.g. in the case of customers.

**Comment by xalien10** [comment]
Actually that was removed in few services in pyfortnox but it still exists in most of the services. I'll fixed it in the next release. Thanks for your concern.

**Comment by xalien007** [comment]
This problem is fixed in the the release 1.2.0. Please upgrade your package

**Comment by xalien10** [comment]
This problem is fixed in the the release 1.2.0. Please upgrade your package
